compose/docker-compose.yaml

```diff
@@ -59,6 +59,20 @@ services:
       control-plane:
         aliases:
           - ${CONTROL_PLANE_DOCKER:-control-plane-main}
+  keycloak:
+    image: jboss/keycloak
+    environment:
+      KEYCLOAK_USER: admin
+      KEYCLOAK_PASSWORD: p
+      KEYCLOAK_IMPORT: /tmp/import-realm.json -Dkeycloak.profile.feature.upload_scripts=enabled
+    volumes:
+      - ./keycloak-realm.json:/tmp/import-realm.json:z,ro
+    ports:
+      - "8080:8080"
+    networks:
+      control-plane:
+        aliases:
+          - keycloak
```

Comment:
Not needed, compose will always create a service name alias, only use it for additional names (but keep it connected to the `control-plane` network).